### DISCUSSION TITLE
Difference between "Dropdown" and "Dropdown (Multiple)" in Custom Fields

### DISCUSSION THREAD

**Discussion by refugeindhamma** [liedekef/events-made-easy#346] [Q&A] [answered]
Hi Franky,

What's the difference between "Dropdown" and "Dropdown (Multiple)" in Custom Fields?

I wanted to make 2 dropdowns: Adults & Children, so I thought "Dropdown (Multiple)" will be able to do this but it does not seem to be the case. I played around with it but can't figure out how it is different from the regular "Dropdown" field type.

Also can't find anything on it in the documentation. An explanation on how to use it to be added to the documentation will be much appreciated.

**Comment by liedekef** [comment, marked as answer]
"Dropdown (Multiple)" allows to choose multiple values, regular dropdown just one.

**Reply by refugeindhamma** [reply]
I see. Thanks for the prompt reply.
